lib/simply_orm.rb

```diff
@@ -0,0 +1,94 @@
+class SimplyORM
+  def initialize(host='localhost', user='blog', password='vadimblog', dbname='blog_development')
+    @host     = host
+    @user     = user
+    @password = password
+    @dbname   = dbname
```

Comment:
Thanks for the advice. I have a few questions:
1. What can you say about the structure of the class? Do I properly separated class into private and public methods?
2. In the code, I have had a lot of code duplication when I make new methods. How can I make refactoring for methods "define_method_all, ..find, ..where", to fit them all in one method?